packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -821,6 +822,78 @@ void main() {
       binding.postTest();
     });
   });
+
+  group('Accessibility announcements testing API', () {
+    testWidgets('Returns last announcement', (WidgetTester tester) async {
```

Comment:
Similarly, as a sanity check, we could verify that the mock handler is installed, for example:

```dart
expect(checkMockMessageHandler(SystemChannels.accessibility.name, null), isFalse);
```

Reply:
Good point! Added the sanity check to the test.